superset/reports/schemas.py

```diff
@@ -208,10 +209,34 @@ class ReportSchedulePostSchema(Schema):
         dump_default=None,
     )
     force_screenshot = fields.Boolean(dump_default=False)
+    custom_width = fields.Integer(
+        metadata={
+            "description": _("Custom width of the screenshot in pixels"),
+            "example": 1000,
+        },
+        allow_none=True,
+        required=False,
+        default=None,
+    )
+
+    @validates("custom_width")
+    def validate_custom_width(self, value: int) -> None:  # pylint: disable=no-self-use
```

Comment:
can we add the same validation to the UI? With some danger text under the input or sth

Reply:
Good point, we currently show validation errors post-`POST`, but it would be great to show client-side.

Since this is time sensitive  I'm going to do that in a separate PR, though, are you OK with that?